temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
The 'pdb' module is typically used for debugging purposes and should not be included in production code. If this is intentional for debugging, consider removing it before merging.